components/Carousel/CarouselItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ParallaxImage } from 'react-native-snap-carousel';
```

Comment:
Parralax is great to view components inside the carousel. Great job getting the carousel up and running